openedx/core/djangoapps/schedules/README.rst

```diff
@@ -0,0 +1,366 @@
+Dynamic Pacing Schedules
+========================
+
+The Schedules app allows course teams to automatically email learners in
+self-paced courses. The emails are designed to keep learners engaged
+with a course's content. Learners receive these messages at important
+milestones throughout a course.
+
+With Schedules, the author of a self-paced course opts learners into one
+of two "Schedule Experiences". Learners either receive Weekly Course
+Highlight Messages, or a combination of Recurring Nudges and Upgrade
+Reminders.
+
+The app can send all three message types "out of the box":
+
+-  Recurring Nudges
+-  Upgrade Reminders
+-  Weekly Course Highlight Messages
+
+Recurring Nudges encourage learners to engage with self-paced courses at
+regular intervals. The app sends nudges three days and ten days after a
+learner enrolls in a course.
+
+Upgrade Reminders ask learners to purchase their course's "Verified"
+certificate. The reminders are sent two days from their course's upgrade
```

Comment:
nit: to be more clear I might say "The reminders are sent two days before their course's upgrade deadline,"